### PR TITLE
Issue/docker demo

### PR DESCRIPTION
**Pull request by Light4Code**

### CODE REVIEW

**Review by Calnoras — COMMENTED** [review]
looking good. Testing now

File: src/Web/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM mcr.microsoft.com/dotnet/sdk:7.0 AS BUILD
+COPY ["./src", "/src"]
+WORKDIR /src/Web
+RUN rm *.db
```
Comment (Calnoras) [inline src/Web/Dockerfile:4]: `RUN if test -n "$(find . -maxdepth 1 -name '*.db' -print -quit)"; then rm *.db; fi `

when no db is given then the dockerfile will fail

**Review by Calnoras — CHANGES_REQUESTED** [review]
linux tests okay. 
windows tests okay (just painfully slow :D )
only issue i found was with the db removal that lead to an error when not present

File: src/Web/Dockerfile (hunk 1 above)
Comment (Light4Code) [reply src/Web/Dockerfile]: Thanks for the finding, fixed it.

**Pull request merged**